web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -1,281 +1,347 @@
 <script lang="ts">
 
-import { movementSensorApi as movementsensorApi, type ServiceError, type commonApi } from '@viamrobotics/sdk';
```

Comment:
not sure what's going on with this diff. trying to figure out how to display my changes only. Sorry about this